client/extensions/wp-job-manager/components/settings/index.jsx

```diff
@@ -0,0 +1,68 @@
+/**
+ * External dependencies
+ */
+import React, { cloneElement, Children, Component, PropTypes } from 'react';
+import { connect } from 'react-redux';
+import { localize } from 'i18n-calypso';
+import { flowRight } from 'lodash';
+
+/**
+ * Internal dependencies
+ */
+import DocumentHead from 'components/data/document-head';
+import Main from 'components/main';
+import Navigation from '../navigation';
+import QuerySettings from '../data/query-settings';
+import { getSelectedSite, getSelectedSiteId } from 'state/ui/selectors';
+import { getSettings } from '../../state/settings/selectors';
+
+class Settings extends Component {
+	static propTypes = {
+		children: PropTypes.element,
+		initialValues: PropTypes.object,
+		site: PropTypes.object,
+		siteId: PropTypes.number,
+		tab: PropTypes.string,
+		translate: PropTypes.func,
+	};
+
+	render() {
+		const {
+			children,
+			initialValues,
+			site,
+			siteId,
+			tab,
+			translate,
+		} = this.props;
+		const mainClassName = 'wp-job-manager__main';
+
+		return (
+			<Main className={ mainClassName }>
+				<QuerySettings siteId={ siteId } />
+				<DocumentHead title={ translate( 'WP Job Manager' ) } />
+				<Navigation activeTab={ tab } site={ site } />
+				{
+					Children.map( children, child => cloneElement( child, { initialValues } ) )
+				}
+			</Main>
+		);
+	}
+}
+
+const connectComponent = connect(
+	( state ) => {
+		const siteId = getSelectedSiteId( state );
+
+		return {
+			initialValues: getSettings( state, siteId ),
+			site: getSelectedSite( state ),
+			siteId,
```

Comment:
Nit -- It's a bit ugly to need both `site` and `siteId` from within the same component. `site` is only needed for `Navigation`, isn't it? Should we directly `connect` `Navigation` to it instead?